package.json

```diff
@@ -19,7 +21,10 @@
 		"watch:webpack": "cross-env NODE_OPTIONS=\"--loader tsx --no-warnings=ExperimentalWarning \" webpack --mode=development --watch"
 	},
 	"stylelint": {
-		"extends": "stylelint-config-xo",
+		"extends": [
+			"stylelint-config-xo",
+			"stylelint-config-prettier"
```

Comment:
Sounds like we don't need this if we update our stylelint config in preparation of their next release.

Reply:
Update what? We're already using the latest versions 

https://github.com/xojs/stylelint-config-xo/issues/19

Reply:
Ah, we're bringing it in via XO, nevermind.